### ISSUE TITLE
'Reset Configuration' breaks Modules Menu in Preferences

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#5283]
### Bug description

If a User resets the configuration of Audacity via 'Tools -> Reset Configuration', and opens the Modules menu in Preferences, the User will observe all of the modules to be missing. 

### Steps to reproduce

1. Launch Audacity
2. Navigate to 'Preferences:Modules' menu via Preferences
3. Verify that modules are present within this menu
4. Reset Audacity confugration via 'Tools -> Reset Configuration'
5. Navigate to 'Preferences:Modules' menu via Preferences
6. Observe Result

### Expected behavior

Modules are still present if the User resets Audacity configuration via 'Tools -> Reset Configuration'

### Actual behavior

Modules are not present if the User resets Audacity configuration via 'Tools -> Reset Configuration'

### Audacity Version

latest master

### Operating system

Windows, macOS

**Comment by chinakov** [comment]
It is also reproduced on 3.2.5 version of Audacity.

**Comment by chinakov** [comment]
Bisected the issue to the original New Export UI Commit. 

https://github.com/audacity/audacity/commit/d5b06017e2ac9f0e71277fb3c9e50853fff50800

**Comment by chinakov** [comment]
I stand corrected by @vsverchinsky - turns out the bug is present in much older versions of Audacity, as I've managed to reproduce it on very old versions of Audacity. 

So, I've traced it back to the 2.4.2 Release, where the **'Reset Configuration'** button was introduced.
**Modules** were introduced in earlier versions of Audacity. 

Which makes me think that this could have been wanted behaviour, but with the new Export UI 'Modules' page in Preferences - this behaviour may be changed now.
